test-suite/src/test/scala/org/scalajs/testsuite/javalib/ConcurrentHashMapTest.scala

```diff
@@ -0,0 +1,110 @@
+package org.scalajs.testsuite.javalib
+
+import language.implicitConversions
+
+import scala.language.implicitConversions
+
+import org.scalajs.jasminetest.JasmineTest
+
+object ConcurrentHashMapTest extends JasmineTest {
+
+  describe("java.util.concurrent.ConcurrentHashMap") {
+
+    it("should store strings") {
+      val chm = new java.util.concurrent.ConcurrentHashMap[String, String]()
+      assert ( chm != null )
```

Comment:
Have a little faith in our compiler. Don't assert that the result of a `new` is not null. This is guaranteed by the language. So just remove this assert.